src/iv/imageviewer.cpp

```diff
@@ -854,14 +854,11 @@ ImageViewer::open()
         if (filename.empty())
             continue;
         add_image(filename);
-        //        int n = m_images.size()-1;
-        //        IvImage *newimage = m_images[n];
-        //        newimage->read_iv (0, false, image_progress_callback, this);
```

Comment:
not sure if this and all the other commented parts of the code are there for a reason, so feel free to ask to revert if you believe it is needed